crates/client/rpc-core/src/lib.rs

```diff
@@ -17,11 +17,13 @@ use hex::ToHex;
 use jsonrpsee::core::RpcResult;
 use jsonrpsee::proc_macros::rpc;
 use mp_starknet::execution::types::{ContractClassWrapper, EntryPointTypeWrapper, EntryPointWrapper, MaxEntryPoints};
+use mp_starknet::transaction::types::Transaction;
 use serde::{Deserialize, Serialize};
 
 pub type FieldElement = String;
 pub type BlockNumber = u64;
 pub type BlockHash = FieldElement;
+pub type TransactionHash = FieldElement;
```

Comment:
Could it be better to propose a `pub type Hash = FieldElement` to not have `TransactionHash`, `BlockHash`, etc...?